pyiron_base/generic/filedata.py

```diff
@@ -83,17 +109,21 @@ def _load_default(file):
         filetype = '.' + filetype
 
     if filetype.lower() in ['.h5', '.hdf']:
-        return FileHDFio(file_name=filename)
+        if project is None:
```

Comment:
Good point! I am happy to force everyone who wants to use this function directly to provide an actual `Project`.

Reply:
Well, I just recognized the `FileData` class directly below calls this without a `Project` 😅. And I actually like to have an `ProjectHDFio` if possible, since otherwise a `__getitem__` of a path super to the root from this object would provide a `pyiron_base.Project`. On the other hand, I doubt that this function will be called on `hdf` files frequently since all 'our' hdf files are handled natively and excluded from `list_nodes()`...